packages/contracts-bedrock/src/L2/L2StandardBridgeInterop.sol

```diff
@@ -40,9 +40,9 @@ contract L2StandardBridgeInterop is L2StandardBridge {
     event Converted(address indexed from, address indexed to, address indexed caller, uint256 amount);
 
     /// @notice Semantic version.
-    /// @custom:semver +interop
+    /// @custom:semver +interop-beta.1
     function version() public pure override returns (string memory) {
-        return string.concat(super.version(), "+interop");
+        return string.concat(super.version(), "+interop-beta.1");
```

Comment:
Now I'm not very sure about this. Maybe we can ask the op team how to handle versioning on this cases, when we are updating a contract that was never released under any version, but that differs from the latest one

Reply:
Solved here https://discord.com/channels/1244729134312198194/1244736279384686614/1288553525323890760